extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -39,13 +120,17 @@
   height: 100% !important;
   border-radius: 0.5em !important;
   position: absolute !important;
+  z-index: 1 !important;
 }
 
 .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
   -webkit-appearance: none !important;
   width: 1.5em !important;
   height: 1.5em !important;
   background: var(--interactive-accent-color) !important;
-  border-radius: 50%;
-  cursor: pointer !important;
+  border-radius: 50% !important;
+}
+
+.i-amphtml-story-interactive-slider-input[disabled] {
+  cursor: initial !important;
```

Comment:
`pointer-events: none` is better, because we want the component to not be clickable after you interacted with it.